### PR TITLE
feat: photos

### PR DESCRIPTION
**Pull request by willruggiano**

### CODE REVIEW

File: schema/application/resolvers/Checklist.ts [hunk 1]
```diff
@@ -122,23 +223,140 @@ export const Checklist: ChecklistResolvers = {
...
+      defaultLimit: Number(
+        process.env.DEFAULT_ATTACHMENT_PAGINATION_LIMIT ?? 20,
+      ),
+      maxLimit: Number(process.env.MAX_ATTACHMENT_PAGINATION_LIMIT ?? 20),
+    });
+
+    // Our (default) order clause specifies:
+    // 1. workresultorder ASC
+    // 2. workresultid ASC
+    const cmp = direction === "forward" ? sql`>` : sql`<`;
```
Comment (jmurphy4) [inline schema/application/resolvers/Checklist.ts:238]: so we are sorting from the API now?

File: schema/system/scalars/index.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
 import type { GraphQLScalarType } from "graphql";
 import {
-  GraphQLDuration,
+  // GraphQLDuration,
```
Comment (jmurphy4) [inline schema/system/scalars/index.ts:3]: are we leaving this for a later date?

**Review by jmurphy4 — APPROVED** [review]

File: schema/system/scalars/index.ts (hunk 2 above)
Comment (willruggiano) [reply schema/system/scalars/index.ts:3]: Potentially yeah. It's a different format (ISO Duration) than what we do for tat right now (seconds) so can't use it

File: schema/application/resolvers/Checklist.ts (hunk 1 above)
Comment (willruggiano) [reply schema/application/resolvers/Checklist.ts:238]: Always have been. This just makes it paginated.

**Pull request merged**